api/mailbox/mailbox.spec.js

```diff
@@ -0,0 +1,52 @@
+const app = require('../../app');
+
+const expect = require('chai').expect;
+const request = require('supertest');
+
+const mailboxDao = require('../../dao/mailbox/');
+
+
+describe('/mailbox api', () => {
+  let id;
+  it('should create a new mailbox', (done) => {
+    request(app)
```

Comment:
Add pre-run assertions for all test cases.

Example: We also have to assert that the circle wasn't existing before calling the create-circle API.